### DISCUSSION TITLE
自动检测更新的小问题

### DISCUSSION THREAD

**Discussion by lyppiao** [ywmoyue/biliuwp-lite#554] [General]
反馈一个小问题，经常使用测试版的，会偶尔遇到一种情况。
假设最新正式版为1.1，更新了测试版1.1.1后，会提醒你不是最新版，可以下载1.1。
点个忽略当前更新就行了，很简单，不修也没关系。

**Comment by ywmoyue** [comment]
关于这个事情，我本来设想的是临时版本的版本号应该是下一个正式版版本号的第四位不为0，比如当前版本1.1.1的话，后面发临时版本应该是1.1.2.x，但是后来发现这样会导致uwp中1.1.2.x比1.1.2版本高不能升级，所以现在发临时版本取当前正式版本的第四位不为0了，然后之前写的检测更新规则也一直没改
